src/utils/drupalClient.ts

```diff
@@ -3,8 +3,8 @@ import crossFetch from 'cross-fetch'
 import { SocksProxyAgent } from 'socks-proxy-agent'
 import { addCAs } from 'syswide-cas'
 
-const baseUrl = process.env.NEXT_PUBLIC_DRUPAL_BASE_URL
-const useProxy = baseUrl.includes('cms.va.gov')
+export const baseUrl = process.env.NEXT_PUBLIC_DRUPAL_BASE_URL || 'cms.va.gov'
+export const useProxy = baseUrl.includes('cms.va.gov')
```

Comment:
For some reason the unit test was failing without me adding an alternative url, I just put this in as a placeholder